modules/phenyl-interfaces/test-cases/index.js

```diff
@@ -259,6 +283,32 @@ export const assertEntityClient = (
         })
       })
 
+      it ('rename entities with updateMulti command', async () => {
+        const result = await entityClient.updateMulti({
+          entityName: 'user',
+          where: { 'name.last': 'Tanaka' },
+          operation: {
+            $rename: {
+              hobbies: 'skills',
+              'favorites.music.singer': 'writer',
+            }
+          },
+        })
+
+        assert(result.ok === 1)
+        assert(result.n === 7)
+
+        const result2 = await entityClient.find({
+          entityName: 'user',
+          where: { 'name.last': 'Tanaka' },
+        })
+
+        result2.entities.forEach(entity => {
```

Comment:
This test could pass if `result2` is empty array.

Putting the following assertion can check it.
```js
assert(result2.length > 0)
```

Reply:
Thank you for your pointing out.
I added assertions.